src/components/Inventory/view/BatchesCellRenderer.tsx

```diff
@@ -19,15 +20,18 @@ export default function BatchesCellRenderer(props: RendererProps<TableRowType>):
   const classes = useStyles();
   const { column, row, value, index, onRowClick } = props;
 
-  const withdraw = () => {
+  const rowClick = (event?: React.SyntheticEvent) => {
```

Comment:
Oh, this was easy! thanks for doing it! I tried with MouseEvent and e.preventDefault and didn't work!